src/demo/components/TUserInfo.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from "react";
+import "./TUserInfo.css";
+import { api, ExternalUser, SdkError, SdkErrors } from "@tonomy/tonomy-id-sdk";
+import { useNavigate } from "react-router-dom";
+import { TH3, TP } from "../../common/atoms/THeadings";
+import "@tonomy/tonomy-id-sdk/build/api/tonomy.css";
+import { TButton } from "../../common/atoms/TButton";
+
+export default function TPageLeftView() {
+  const [user, setUser] = useState<ExternalUser | null>(null);
+  const [accountName, setAccountName] = useState<string>("");
+  const [username, setUsername] = useState<string>("");
+  const navigation = useNavigate();
+
+  async function onRender() {
+    try {
+      const user = await api.ExternalUser.getUser();
```

Comment:
get this from userStore. Then you do not need to do the handling catches

Reply:
in demo we don't have the userStore can i use it from sso?

Reply:
still not resolved